Chapter1/BMI_Calculator.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter1
+{
+
+    public static class BMICalculator
+    {
+        public static void BMI_Calculator()
+        {
+            // Person 1 
+
+            Console.Write("Can I have your name: ");
+            var name = Console.ReadLine();
+            Console.Write("Can I have your surname: ");
+            var surname = Console.ReadLine();
+            Console.Write("Can I have your age:");
+            var age = Console.ReadLine();
+            Console.Write("Can I have your weight in kg: ");
+            var weight = Console.ReadLine();
+            Console.Write("Can I have your height in cm: ");
+            var height = Console.ReadLine();
+
+            Console.WriteLine($"{name} {surname} is {age} years old, his weight is {weight} kg and his height is {height} cm.");
+
+            var BMI = calculate(height, weight);
+            Console.WriteLine($"This person bmi is {BMI}");
+
+            // Person 2 
```

Comment:
While this works I feel like it might be against the spirit of the exercise. Can you find a way to do the same without duplicating your code?

Reply:
yep, I can use a for loop. Did not know I was allowed to use loops